buckets.yaml

```diff
@@ -0,0 +1,17 @@
+# A mapping of bucket URLs to bucket metadata
+#
+# Possible bucket metadata:
+#   prefix: the identifier used for this bucket on the test-history dashboard.
+#   sequential: an optional boolean that indicates whether test runs in this
+#     bucket are numbered sequentially. This is used for an optimization in the
+#     test-history collection phase, and defaults to true.
+
+gs://kubernetes-jenkins/logs/:
```

Comment:
I don't think this is the right layer of abstraction. The key should be the federated source